eng/common/pipelines/templates/jobs/perf.yml

```diff
@@ -13,7 +13,7 @@ parameters:
   default: 'azsdk-pool-mms-win-2019-perf'
 - name: WindowsVmImage
   type: string
-  default: 'MMS2019'
+  default: 'MMS2019Compliant'
```

Comment:
Cannot make eng/common changes in the language repo.

Reply:
Yeah, I forgot to check for that after I did textual search and replace. I will revert. Thanks!